Sources/SwiftGRPCNIO/RawGRPCToGRPCCodec.swift

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import NIO
+import NIOHTTP1
+import SwiftProtobuf
+
+/// Outgoing gRPC package with a fixed message type.
+public enum GRPCClientRequestPart<MessageType: Message> {
```

Comment:
`RequestMessage` might make more sense than `MessageType` (and `ResponseMessage` below)

Reply:
True. This is just a mirror of `GRPCServer{Request,Response}Part`; I'll update them all.